### ISSUE TITLE
Feature Request: Allow Specific User Selection

### ISSUE THREAD

**Issue by RazKissos** [login-securite/DonPAPI#75]
### Background

This is less of a bug and more of a "program behavior" request.  
As I run `DonPAPI` on a target, I encounter many junk accounts that exist on the remote machine.'s `Users` folder.  
This was partially mitigated with the new "false positive" folders in each `triage class`.  
However, usually when getting loot from a machine, there are more junk folders than can be expected to deal with.  
This is an issue since we cannot expect to deal with every possible "false positive" we encounter.  
So, I suggest an addition of a user selection prompt / flag, which will allow users to select specific windows users for enumeration.  

#### Interactive prompting

> This is obviously not ideal when we want to iterate over many machines but it might be useful to add.

```log
Found 5 User Folders (Bob, Alice, John Doe, Trash), please select one or more (delimited by '^'):
> Bob^John Doe
```

#### Startup Parameter

```bash
donpapi -u admin -p admin -t 1.2.3.4 --windows-users "Bob" "John Doe"
```

### Summary

This is obviously not an urgent matter but it is certainly a "nice to have" feature.  
Thank you for reading, I wish you all the best!

**Comment by zblurx** [comment]
Hi, thank you for the feature idea. I don't like the idea of a selection prompt, because like you said, when running on 500+ target, it can be annoying. Also, users home directory name can be inconsistent between multiple computers : sometime it is `user` or sometimes `user.domain` or even `user.workstation_name`, so hard to add a functional whitelist. 
Maybe we could add a blacklist, in order to add multiple false positive, but I also think about a `--only-domain-user` to target only user that seems to be domain joined. What do you think ?

**Comment by RKivy** [comment]
I don't think there is any difference between a blacklist and a whitelist - this is because in either case we predict an unknown folder name, the only difference is that when using a whitelist of allowed folders we can limit the gathering to a specific user on the machine.  
This is especially useful for when you have a **Terminal Server** that many users use to RDP to and we only really want the domain users that we find interesting.  
This will allow donpapi to be used for more "surgical" looting as opposed to a spray, extending the use-cases and speed of execution.

**Comment by Touff** [comment]
it could be done fairly quickly @covertivy 
add a flag in entry.py, and then append/whitelist your users in the 
@property
    def users(self) -> List[str]:
around the : false_positive = 

might be time for your first PR :-)

**Comment by covertivy** [comment]
Thought I'd let you know - the code is driving me insane.
The `false_positive` variable is not even used - it is hard coded in the dploot codebase.
[I created a PR](https://github.com/zblurx/dploot/pull/30) to fix the issues but it will be possible only after it is committed an pushed to master on the dploot project itself.
Honestly both codebases are very messy and need to be worked on...

**Comment by covertivy** [comment]
As you can tell - this issue is related to [this pull request](https://github.com/login-securite/DonPAPI/pull/119) I created to address it.  

I would be extremely glad if you merge it 😄 

Thanks @zblurx !

**Comment by covertivy** [comment]
@login-securite @zblurx please let me know when will the [PR](https://github.com/login-securite/DonPAPI/pull/119) be merged.
